Sources/Networking/Responses/OfferingsResponse.swift

```diff
@@ -18,12 +18,22 @@ import Foundation
 
 public struct PaywallComponentsData: Codable, Equatable, Sendable {
 
+    public struct ComponentsConfigs: Codable, Equatable, Sendable {
+
+        public var base: ComponentsConfig
+
+        public init(base: ComponentsConfig) {
+            self.base = base
+        }
+
+    }
+
     public struct ComponentsConfig: Codable, Equatable, Sendable {
 
-        public var components: [PaywallComponent]
+        public var stack: PaywallComponent.StackComponent
 
-        public init(components: [PaywallComponent]) {
-            self.components = components
+        public init(stack: PaywallComponent.StackComponent) {
+            self.stack = stack
```

Comment:
This was the big change where a paywall _was_ just a list of components but now its a stack.

Reply:
Like I said, clean!